.github/workflows/build_and_push.yaml

```diff
@@ -45,4 +47,5 @@ jobs:
           cache-to: type=inline
           tags: |
             lemonsaurus/blackbox:latest
-            lemonsaurus/blackbox:${{ steps.sha_tag.outputs.tag }}
+            lemonsaurus/blackbox:${{ steps.tags_generator.outputs.tag }}
+            lemonsaurus/blackbox:${{ steps.tags_generator.outputs.semver }}
```

Comment:
This looks like the right approach, but this workflow executes on push to main, which makes it inaccurate. We can't push a semver-tagged version to DockerHub every time there's a master commit - We have to push them to DockerHub when we actually _release_ a new version, via the GitHub Release mechanism.